sims/verilator/Makefile

```diff
@@ -40,6 +40,31 @@ debug: $(sim_debug)
 include $(base_dir)/common.mk
 include $(sim_dir)/verilator.mk
 
+#########################################################################################
```

Comment:
Tried to model how the VCS makefile does things. I think I did most of the split correctly (CC v non-CC).